src/main/java/com/zeepy/server/community/service/CommunityService.java

```diff
@@ -195,5 +200,34 @@ public void updateCommunity(Long communityId, UpdateCommunityReqDto updateCommun
 			.orElseThrow(NotFoundCommunityException::new);
 		updateCommunityReqDto.updateCommunity(findCommunity);
 	}
+
+	@Transactional(readOnly = true)
+	public CommunityResponseDto getCommunity(Long communityId) {
+		Community community = communityRepository.findById(communityId)
+			.orElseThrow(NotFoundCommunityException::new);
+		return new CommunityResponseDto(community);
+	}
+
+	@Transactional(readOnly = true)
+	public Page<CommunityResponseDto> getCommunityList(String address, String communityType, Pageable pageable) {
+		Page<Community> communityList;
+		if (address == null) {
+			if (communityType == null) {
+				communityList = communityRepository.findAll(pageable);
+			}
+			communityList = communityRepository.findByCategory(CommunityCategory.valueOf(communityType), pageable);
+		}
+
+		if (communityType == null) {
+			communityList = communityRepository.findByAddress(address, pageable);
+		}
```

Comment:
address랑 communityType이 빈 스트링(empty String)인지도 한번 확인해야 하지 않을까요? 
```suggestion
		Page<Community> communityList;
		if (address == null || address.isEmpty()) {
			if (communityType == null || commnityType.isEmpty()) {
				communityList = communityRepository.findAll(pageable);
			}
			communityList = communityRepository.findByCategory(CommunityCategory.valueOf(communityType), pageable);
		}

		if (communityType == null || communityType.isEmpty()) {
			communityList = communityRepository.findByAddress(address, pageable);
		}
```